### PR TITLE
SAIC-779 Fix FT to ignore deleted tenant's resources.

### PR DESCRIPTION
**Pull request by plomakin**
Included is_tenant deleted checks.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.test_vm_migration
devlab/tests/test_vm_migration.py:3: [W0403(relative-import), ] Relative import 'functional_test', should be 'tests.functional_test'
devlab/tests/test_vm_migration.py:4: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
************* Module tests.functional_test
devlab/tests/functional_test.py:15: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/functional_test.py:20: [W0403(relative-import), ] Relative import 'generate_load', should be 'tests.generate_load'
devlab/tests/functional_test.py:21: [W0403(relative-import), ] Relative import 'utils', should be 'tests.utils'
devlab/tests/functional_test.py:29: [E0401(import-error), ] Unable to import 'cfglib'
devlab/tests/functional_test.py:160: [W0106(expression-not-assigned), FunctionalTest.filter_volumes] Expression "[volumes.extend(i['cinder_volumes']) for i in config.tenants if ('cinder_volumes' in i) and (not i.get('deleted'))]" is assigned to nothing
devlab/tests/functional_test.py:184: [W0106(expression-not-assigned), FunctionalTest.get_vms_with_fip_associated] Expression "[vms.extend(i['vms']) for i in config.tenants if 'vms' in i]" is assigned to nothing
************* Module tests.utils
devlab/tests/utils.py:17: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/utils.py:100: [W0621(redefined-outer-name), MigrationUtils.__init__] Redefining name 'config' from outer scope (line 17)
devlab/tests/utils.py:111: [E1129(not-context-manager), MigrationUtils.execute_command_on_vm] Context manager 'generator' doesn't implement __enter__ and __exit__.

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: devlab/tests/functional_test.py [hunk 1]
```diff
@@ -182,3 +183,10 @@ def get_vms_with_fip_associated(self):
         vms = config.vms
         [vms.extend(i['vms']) for i in config.tenants if 'vms' in i]
         return [vm['name'] for vm in vms if vm.get('fip')]
+
+    def is_tenant_exist(self, keystone_client, tenant_id):
```
Comment (ntymtsiv) [inline devlab/tests/functional_test.py]: We have similiar method in generate_load (https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/devlab/tests/generate_load.py#L249)
Please remove one of them.

Comment (plomakin) [inline devlab/tests/functional_test.py]: The problem is that we don't have such method for tenant_id, and here we can't get tenant_name for vms we've got from SRC cloud. So, i'll prefer to keep that method as well as method in generate_load.

Comment (ntymtsiv) [inline devlab/tests/functional_test.py]: Ok, i see

**Comment by ntymtsiv** [comment]
+1, Looks good

**Comment by plomakin** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.test_vm_migration
devlab/tests/test_vm_migration.py:3: [W0403(relative-import), ] Relative import 'functional_test', should be 'tests.functional_test'
devlab/tests/test_vm_migration.py:4: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
************* Module tests.functional_test
devlab/tests/functional_test.py:15: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/functional_test.py:20: [W0403(relative-import), ] Relative import 'generate_load', should be 'tests.generate_load'
devlab/tests/functional_test.py:21: [W0403(relative-import), ] Relative import 'utils', should be 'tests.utils'
devlab/tests/functional_test.py:29: [E0401(import-error), ] Unable to import 'cfglib'
devlab/tests/functional_test.py:160: [W0106(expression-not-assigned), FunctionalTest.filter_volumes] Expression "[volumes.extend(i['cinder_volumes']) for i in config.tenants if ('cinder_volumes' in i) and (not i.get('deleted'))]" is assigned to nothing
devlab/tests/functional_test.py:184: [W0106(expression-not-assigned), FunctionalTest.get_vms_with_fip_associated] Expression "[vms.extend(i['vms']) for i in config.tenants if 'vms' in i]" is assigned to nothing
************* Module tests.utils
devlab/tests/utils.py:17: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/utils.py:100: [W0621(redefined-outer-name), MigrationUtils.__init__] Redefining name 'config' from outer scope (line 17)
devlab/tests/utils.py:111: [E1129(not-context-manager), MigrationUtils.execute_command_on_vm] Context manager 'generator' doesn't implement __enter__ and __exit__.

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by pavloivanets** [comment]
+1 Looks OK

File: devlab/tests/functional_test.py (hunk 1 above)
Comment (roman-verchikov) [inline devlab/tests/functional_test.py]: Please rename to `tenant_exists()`

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**